src/app/pages/home/about/about.component.html

```diff
@@ -1 +1,41 @@
-<div>About Page</div>
\ No newline at end of file
+<div class="aboutus-overlay1">
```

Comment:
Suggest we wrap this page using Bootstrap's `container` class instead of our own custom class. This should take care of left/right spacing as well.
```suggestion
<div class="container">
```
You can set the component itself to take the entire browser's height and set its content to vertically center themselves using the following stlyes:

```scss
:host {
    height: 100vh;
    display: flex;
    flex-direction: column;

    & > .container {
        margin: auto;
    }
}
```